nss-day-cohort-43/nuttyshell-sons-of-witches
# Bs adding functionality to add friend button

# Description
Added friends list container with add friend functionality. Delete functionality is also partially functional but needs troubleshooting (deletes all friends rather than only the selected one).

Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Added friends list container. Please ensure you are able to add a new friend by entering the friend's username and email address and then clicking the Save button.

# Checklist:
